cityguide/source/app/src/main/java/org/kaaproject/kaa/demo/cityguide/fragment/PlacesFragment.java

```diff
@@ -119,7 +119,7 @@ public void onEvent(Events.ConfigurationUpdated configurationUpdated) {
 
     @Override
     public String getTitle() {
-        return mAreaName + "_" + mCityName;
```

Comment:
@maxml Sorry, but I don't think that user will have any problems with understanding on page he is, because he can only get to the city page after choosing the country before. The flow is next: Country --> City --> Place.